NEWS

```diff
@@ -35,16 +35,16 @@ CHANGES WITH 257 in spe:
           Accept=yes sockets too, where it was previously silently ignored and
           "connection" was used unconditionally.
 
-        * systemd-logind now always obeys inhibitor locks, where previously it
-          ignored locks taken by the caller or when the caller was root. A
+        * systemd-logind now always obeys block inhibitor locks, where previously
+          it ignored locks taken by the caller or when the caller was root. A
```

Comment:
"the caller" is a bit vague and lacks context?

Reply:
It seems clear to me - anyway it's just the NEWS entry, you can edit it yourself later if you have thoughts on how to make it clearer, it doesn't need to block RC1